Feature/170 remove role based authorization plugin on setup (#173)
cloudogu/jenkins: docs/gui/release_notes_en.md

Technical details on a release can be found in the corresponding [Changelog](https://docs.cloudogu.com/en/docs/dogus/jenkins/CHANGELOG/).

## Release 2.452.4-3
- With /var/lib/jenkins/init.groovy.d/plugin-blocklist.json, the Dogu offers a block list for plugins in Jenkins.

>     - Attempts to install the plugin result in an automatic removal of the plugins listed in the block list

Reviewer: ```suggestion
    - Attempts to install plugins from the block list will result in an automatic removal after a restart
```
